operator/src/main/java/oracle/kubernetes/operator/helpers/CallBuilder.java

```diff
@@ -594,6 +600,40 @@ public VersionInfo readVersionCode() throws ApiException {
     return DISPATCHER.execute(factory, requestParams, helper);
   }
 
+  /**
+   * Execute a synchronous call with a retry on failure.
+   * @param call The call
+   * @param retryDelaySeconds Retry delay in seconds
+   * @param <T> Call return type
+   * @return Results of operation, if successful
+   * @throws Exception Exception types other than ApiException, which will cause failure
+   */
+  public <T> T executeSynchronousCallWithRetry(Callable<T> call, int retryDelaySeconds) throws Exception {
+    T result = null;
+    boolean complete = false;
+    do {
+      try {
+        result = call.call();
+        complete = true;
+      } catch (RuntimeException re) {
```

Comment:
What happened if it is a RuntimeException but not an ApiException?  Do we want to limit the number of retries or make it retry forever (hopefully the condition can be resolved)?

Reply:
I'll think about other exceptions... I didn't want to limit the number of retries here because the operator cannot go on until it can connect to the master and the operator will eventually be killed by the liveness probe.